backend/sites/src/app/entities/siteSubdivisions.entity.ts

```diff
@@ -63,7 +70,7 @@ export class SiteSubdivisions extends ChangeAuditEntity {
   sprofDateCompleted: Date | null;
 
   @Field()
-  @Column('bigint', { primary: true, name: 'site_subdiv_id' })
+  @PrimaryGeneratedColumn('increment')
```

Comment:
is this a new primary column? if not, this might create sync issue if siteSubdivId was not set as auto generated. Besides, will the column name be 'site_subdiv_id' ?

Reply:
These are both the existing primary columns for these tables.

Reply:
The only doubt I have here is with respect to data migration from oracle to postgres. Since oracle already have values for this column, and we make this column auto generated in postgres, whether this would cause an error for the migration

Reply:
Also you might have to add the property  name: 'site_subdiv_id'  to the PrimaryGeneratedColumn

Reply:
I'll double check the column name because I didn't test updating with this, only adding. Thanks for calling that out.

Regarding the data import: I'm concerned about it as well. I don't know about oracle databases, so I don't know how it assigns primary key values. In postgres you typically use a sequence generator like I've defined here. These sequences aren't in the database dump that you provided me. If the oracle database uses sequences like postgres, and ora2pg will do a two way sync of the sequences then this isn't actually an issue.

Is there any way for us to get access to the oracle db to check and also verify the behaviour of ora2pg?

Reply:
Actually, I think the best thing to do here is to wait until the oracle db import is complete, dump the production db and use that to build off of. I'll just continue doing speculative work until that is complete.

Reply:
@bolny , you can raise a SD ticket with approval from Kerri to have readonly access to your IDIR to prod database. Let me know if you need help.